app/src/main/java/edu/uc/reedws/musiclink/ui/main/MainFragment.kt

```diff
@@ -27,4 +23,8 @@ class MainFragment : Fragment() {
         // TODO: Use the ViewModel
     }
 
+    companion object {
+        fun newInstance() = MainFragment()
+    }
+
```

Comment:
I removed the companion object from the top because I learned from one of the professor's lecture videos (Sprint 1 code review) that companion object should be down below in the class as it is one of the Kotlin coding conventions.